engine/table/src/main/java/io/deephaven/engine/table/impl/UpdateByControl.java

```diff
@@ -19,13 +24,18 @@ public interface UpdateByControl {
     double DEFAULT_MAXIMUM_STATIC_SPARSE_MEMORY_OVERHEAD =
             Configuration.getInstance().getDoubleWithDefault("UpdateByControl.maximumStaticMemoryOverhead", 1.1);
 
-    UpdateByControl DEFAULT = new UpdateByControl() {};
+    static ImmutableUpdateByControl.Builder builder() {
```

Comment:
We should be creating a public version of this interface and returning it instead.